internal/server/data/data.go

```diff
@@ -177,6 +177,9 @@ func setOrg(db *gorm.DB, model any) {
 	}
 
 	org := OrgFromContext(db.Statement.Context)
+	if org == nil {
+		panic("missing org id in context")
+	}
```

Comment:
these panics are programmer errors, and should be obvious from tests. If you remove them, it will still panic, but with a worse message.